scripts/zeus.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#PBS -q q_short
+#PBS -l nodes=3:ppn=4
+#PBS -l walltime=00:02:00
+
+SUPERVISOR_HOSTNAME=`hostname`
+REMOTE_HOSTS=`cat ${PBS_NODEFILE} | uniq | grep -v ${SUPERVISOR_HOSTNAME}`
+
+java -jar supervisor.jar &
+SUPERVISOR_PID=$!
+
+for WORKER_HOST in ${REMOTE_HOSTS}
+do
+    pbsdsh -h ${WORKER_HOST} java -jar worker.jar -Dsupervisor.hostname=${SUPERVISOR_HOSTNAME}
```

Comment:
1. Place config opts just after "java". It won't work now. I had same issue with docker and reproduced locally. Jar name must be the last parameter.
2. Worker hostname is required too:
-Dakka.remote.netty.tcp.hostname=${WORKER_HOST}